packages/council-graphql/src/resolvers/index.ts

```diff
@@ -59,40 +64,67 @@ export const resolvers: Resolvers<CouncilContext> = {
         voters = await VoterModel.getAll({ context });
       }
 
-      if (search) {
-        const candidates: string[] = [];
-
-        voters = await Promise.all(
-          voters.map(async (voter) => {
-            if (!voter) {
-              return;
-            }
+      if (search || votingPowerMin || votingPowerMax) {
```

Comment:
Since we've added a `votingVaults` argument, we should do an `else if` before this to get voters by voting vaults if passed before getting all.

Something like:

```ts
let voters: (Voter | undefined)[] = [];
let votingVaults: (VotingVault | undefined)[] = [];

if (addresses) {
  voters = await VoterModel.getByAddresses({ addresses });
} else if (votingVaultAddresses) {
  voingVaults = await VotingVaultModel.getByAddresses({ address: votingVaultAddresses, context })
  voters = await VoterModel.getByVotingVaults({ votingVaults, context });
} else {
  voters = await VoterModel.getAll({ context });
}
```